jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -98,18 +98,26 @@ def calc_compass(image_wcs, x, y, len_deg_e, len_deg_n):
 
 
 def calc_compass_radius(image_wcs, x, y, radius_px):

```

Comment:
@ejeschke , this function was actually from `ginga`:

https://github.com/ejeschke/ginga/blob/7e61d9772ec1e57c332f1e2c0280d308972640c4/ginga/util/wcs.py#L762-L764

Are you interested to have a similar patch over at `ginga`?

Reply:
That looks like a good change, so sure!

Reply:
OK please see https://github.com/ejeschke/ginga/pull/1102 . Thanks!